project/lib/storage_backends.py

```diff
@@ -268,6 +276,17 @@ def path_join(*args):
         # For local storage, we join paths depending on the OS rules.
         return os.path.join(*args)
 
+    def spacer_data_loc(self, key) -> DataLocation:
+        """ Returns a spacer DataLocation object. """
+        return DataLocation(storage_type='filesystem',
+                            key=self.path(key))
+
+    def exists_full(self, full_key) -> bool:
```

Comment:
Judging from commit 86f4ccbbf9c2ee34e277ffab4ea1704a8872b352, it looks like you meant to remove this `exists_full()` method too. Is that right?

Reply:
Yes. thanks for catching that.